Containers/borgbackup/backupscript.sh

```diff
@@ -35,19 +35,19 @@ done
 
 # Check if target is mountpoint
 if ! mountpoint -q /mnt/borgbackup; then
-    echo "/mnt/borgbackup is not a mountpoint which is not allowed"
+    echo "/mnt/borgbackup is not a mountpoint which is not allowed."
     exit 1
 fi
 
 # Check if target is empty
 if [ "$BORG_MODE" != backup ] && [ "$BORG_MODE" != test ] && ! [ -f "$BORG_BACKUP_DIRECTORY/config" ]; then
-    echo "The repository is empty. cannot perform check or restore."
+    echo "The repository is empty. Cannot perform check or restore."
     exit 1
 fi
 
 # Do not continue if this file exists (needed for simple external blocking)
 if [ -f "$BORG_BACKUP_DIRECTORY/aio-lockfile" ]; then
-    echo "Not continuing because aio-lockfile exists - it seems like a script is externally running which is locking the backup archive."
+    echo "Not continuing because aio-lockfile exists -- it seems like a script is externally running which is locking the backup archive."
```

Comment:
i don't understand this additional -

Reply:
Sry, me being a typography nerd here out of habit. According to my knowledge, the correct punctuation mark to be used here is an (spaced) [en-dash](https://en.wikipedia.org/wiki/Dash#En_dash) (or an unspaced em-dash) and not a [hyphen](https://en.wikipedia.org/wiki/Hyphen). At least in LaTeX `--` is replaced by an en-dash (`–`) (and `---` by an em-dash (`—`)), I assumed in markdown this might be the case as well. If utf-8 is used, probably a en-dash (–) can be used directly as well.

Reply:
I have no idea. English is not my native language

Reply:
```suggestion
    echo "Not continuing because aio-lockfile exists – it seems like a script is externally running which is locking the backup archive."
```